.github/workflows/docs.yml

```diff
@@ -156,17 +156,17 @@ jobs:
       - name: Build and install dependencies
         working-directory: bindings/python
         run: |
-          uv sync --all-groups --all-extras
+          uv sync --group docs
 
       - name: Build Docs
         working-directory: bindings/python
-        run: uv run pdoc -t ./template --output-dir ./docs opendal
+        run: make doc
```

Comment:
Can we avoid introduce `Makefile`? I really hate `Makefile`.

If we do need this, we can use `Justfile`.

Reply:
Ok, remove it now